counterparty-core/counterpartycore/lib/mempool.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import time
+
+from counterpartycore.lib import blocks, config, exceptions, ledger
+from counterpartycore.lib.kickstart import blocks_parser
+
+logger = logging.getLogger(config.LOGGER_NAME)
+
+
+def parse_mempool_transactions(db, raw_tx_list):
+    logger.debug("Parsing mempool block")
```

Comment:
Let's not call it a "mempool block". It's not a block. (I think it was a bad idea for me to have implemented it that way originally.)